app/containers/Send/Send.jsx

```diff
@@ -347,7 +347,7 @@ export default class Send extends React.Component<Props, State> {
       const newState = [...prevState.sendRowDetails]
 
       const objectToClear = newState[index]
-      if (objectToClear.errors[field]) {
+      if (objectToClear.errors && objectToClear.errors[field]) {
```

Comment:
just use lodash get for this. you can pass an array I believe for a case like this where you don't know some of the keys and just use variables

Reply:
Thanks @dvdschwrtz 💪